src/page-modules/departures/details/details.module.css

```diff
@@ -0,0 +1,104 @@
+.container {
+  margin: 0 auto;
+  max-width: var(--maxPageWidth);
+  padding: var(--spacings-xLarge);
+
+  display: grid;
+  grid-template-columns: 2fr 3fr;
+  grid-template-areas:
+    'header header'
+    'serviceJourney map';
+  gap: var(--spacings-small);
+}
+@media (max-width: 650px) {
+  .container {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      'header'
+      'serviceJourney'
+      'map';
+  }
+}
+.headerContainer {
+  grid-area: header;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacings-large);
+}
+.mapContainer {
+  grid-area: map;
+  max-height: 37.5rem;
+  width: 100%;
+  margin: 0 auto;
+}
+.serviceJourneyContainer {
+  grid-area: serviceJourney;
+}
+@media (max-width: 650px) {
+  .serviceJourneyContainer {
+    margin: 0 auto;
```

Comment:
Should this be added for smaller screens? Looks like the content gets too far to the right with this, instead of allowing it to use the full width.

Reply:
Nope, this is wrong. Fixed now 👍